plugins/drivers/raw-exec/plugin-go-grpc/raw_exec/utils.go

```diff
@@ -45,11 +45,12 @@ func createExecutor(w io.Writer, executorConfig *ExecutorConfig) (executor.Execu
 	// TODO
 	bin := "nomad"
 
-	config := &plugin.ClientConfig{
-		Cmd: exec.Command(bin, "executor", string(c)),
-		AllowedProtocols: []plugin.Protocol{
-			plugin.ProtocolNetRPC,
-		},
+	config := &plugin.ClientConfig{}
+	// Set either one of reattachConfig/Cmd fields
+	if reattachConfig != nil {
```

Comment:
Consider if this should be a separate function maybe? This can be done at a later time.